src/cargo/core/shell.rs

```diff
@@ -122,7 +144,9 @@ impl MultiShell {
 
 impl Shell {
     pub fn create(out: Box<Write + Send>, config: ShellConfig) -> Shell {
-        if config.tty && config.color {
+        // Check for cfg!(windows) as colored output on Windows can only be supported when a tty is
+        // present.
+        if !cfg!(windows) || config.tty {
```

Comment:
Perhaps this changed line is the one causing failures? I'm also not 100% sure why the bots are failing